modules/tinymce/CHANGELOG.md

```diff
@@ -1396,8 +1397,6 @@ Full documentation for the version 5 features and changes is available at https:
 
 Developer preview 1
 
-Initial list of features and changes is available at https://tiny.cloud/docs-preview/release-notes/new-features/
```

Comment:
This shouldn't have been removed, as now there's nothing listed there.

- [ ] Revert this change

Reply:
its a 404 dead link

Reply:
Removing it is still not the right solution then... we need to fix the link or add something back so that we don't have a version without any entries. I'd suggest linking to https://www.tiny.cloud/docs/release-notes/release-notes50/#newfeatures instead.